locale/hu.json

```diff
@@ -307,28 +307,28 @@
                 "session_time": "Szerveren töltött idő",
                 "play_time": "Játékidő",
                 "joined": "Csatlakozva",
-                "whitelisted_label": "Whitelisted",
-                "whitelisted_notyet": "not yet",
-                "btn_wl_add": "ADD WL",
-                "btn_wl_remove": "REMOVE WL",
-                "btn_wl_success": "Whitelist status changed.",
-                "log_label": "Log",
-                "log_empty": "No bans/warns found.",
-                "log_ban_count": "%{smart_count} ban |||| %{smart_count} bans",
-                "log_warn_count": "%{smart_count} warn |||| %{smart_count} warns",
-                "log_btn": "DETAILS",
-                "notes_changed": "Player note changed.",
-                "notes_placeholder": "Notes about this player..."
+                "whitelisted_label": "Engedélyezve",
+                "whitelisted_notyet": "Még nem",
+                "btn_wl_add": "Engedélyezés megadása",
+                "btn_wl_remove": "Engedélyezés visszavonása",
+                "btn_wl_success": "Az engedélyezési lista állapota megváltozott.",
+                "log_label": "Napló",
+                "log_empty": "Nem található tiltás/figyelmeztetés.",
+                "log_ban_count": "%{smart_count} kitiltás |||| %{smart_count} kitiltások",
+                "log_warn_count": "%{smart_count} figyelmeztetés |||| %{smart_count} figyelmeztetések",
+                "log_btn": "Részletek",
+                "notes_changed": "Játékos jegyzete változott.",
+                "notes_placeholder": "Megjegyzések ehhez a játékoshoz..."
```

Comment:
Hello! I think we should be more consistent here. If you use `jegyzet`, then use `jegyzet` instead of `megjegyzések`. This way (I think at least) its confusing.

Reply:
Hello! You're right, I fixed all of them